docs/articles_en/openvino-workflow/running-inference/inference-devices-and-modes/cpu-device/performance-hint-and-thread-scheduling.rst

```diff
@@ -63,19 +63,19 @@ the model precision and the ratio of P-cores and E-cores.
-+--------------------------------------+------------------------------------------------------------------------+--------------------------------------------------------------------+
-| ``ov::hint::enable_cpu_pinning``     | No / Not Supported                                                     | Yes except using P-cores and E-cores together                      |
-+--------------------------------------+------------------------------------------------------------------------+--------------------------------------------------------------------+
++--------------------------------------+-----------------------------------------------------------------------+-----------------------------------------------------------------------+
+| Property                             | Windows                                                               | Linux                                                                 |
++======================================+=======================================================================+=======================================================================+
+| ``ov::num_streams``                  | 1                                                                     | 1                                                                     |
++--------------------------------------+-----------------------------------------------------------------------+-----------------------------------------------------------------------+
+| ``ov::inference_num_threads``        | is equal to the number of P-cores or P-cores+E-cores on one numa node | is equal to the number of P-cores or P-cores+E-cores on one numa node |
++--------------------------------------+-----------------------------------------------------------------------+-----------------------------------------------------------------------+
+| ``ov::hint::scheduling_core_type``   | :ref:`Core Type Table of Latency Hint <core_type_latency>`            | :ref:`Core Type Table of Latency Hint <core_type_latency>`            |
++--------------------------------------+-----------------------------------------------------------------------+-----------------------------------------------------------------------+
+| ``ov::hint::enable_hyper_threading`` | No                                                                    | No                                                                    |
++--------------------------------------+-----------------------------------------------------------------------+-----------------------------------------------------------------------+
+| ``ov::hint::enable_cpu_pinning``     | No / Not Supported                                                    | Yes except using P-cores and E-cores together                         |
++--------------------------------------+-----------------------------------------------------------------------+-----------------------------------------------------------------------+
 
```

Comment:
I think it would be benefitial to have dedicated subsection here which will describe that starting from 5th Xeon generation Numa node are exposed explicitly (witch SNC=ON) (ideally should be reference on Intel resource with details) and that OV uses only single Numa due to performance consdirations. So usage of only part of the socket cores is expected behavior.
We also need to mentioned that for some models (with big compute demand) default behavior might not be optimal so the recomendation is to try full socket or even multi-socket execution for latency + provide recommendation hot to do that.

Reply:
@dmitry-gorokhov updated. Please take a look.